Fix leftovers in bookmarks - Closes #2206

slaweet opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

### What issue have I solved?
<!--- Complementary description if needed -->
#2206

### How have I implemented/fixed it?
<!--- Describe your technical implementation -->
- [x] Prevent Bookmarks header from jumping when go to Add a new bookmark. By creating a toolbox `PageHeader` component and use it in both pages.
- [x] 'Filter by name' is wrong because filtering works both for name and address - restrict to name only
- [x] Prevent while editing label to save with empty string by disabling the `Save changes` button

### How has this been tested?
<!--- Please describe how you tested your changes. -->
- Go from bookmarks page to Add a new bookmark page
- Try filtering by address
- Try saving with empty title

### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

osvaldovega commented on src/components/toolbox/pageHeader/index.js:
```diff
@@ -0,0 +1,18 @@
+    <span>
+      <h1>{title}</h1>
+      <div className={styles.subtitle}>{subtitle}</div>
+    </span>
```
hey Vit I was checking this and I know that works, and it is not a big deal, will not be better to wrap the title and subtitle in the div tag and then the subtitle to be a span?

slaweet replied:
What I was trying to express is that there are multiple `span`s in a row, but `div` has its own row in its container.  But I don't have a strong opinion about this. So I changed it to your suggestion in https://github.com/LiskHQ/lisk-hub/pull/2225/commits/a0e2be96cf87f1e8f509a18b4f00ef739b6bd733

slaweet commented:
- updated filter placeholder to say '... and address ...', which made it longer then the width of the input, so I also increased the width to 200px
- updated edit placeholder to be same as in https://projects.invisionapp.com/d/main?origin=v7#/console/17570736/368615414/preview
